src/player.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Sprite } from 'react-game-kit';
+
+class Player extends React.Component {
+
+    constructor(props) {
+        super(props);
+        const id = props.id || 0;
+        this.state = {
+            id: id,
```

Comment:
you can just write id: props.id || 0 to be more consistent with the rest of the state